starter_templates/ruby/app/main.rb

```diff
@@ -0,0 +1,16 @@
+# You can use print statements as follows for debugging, they'll be visible when running tests.
```

Comment:
Let's switch the order of statements a bit here. Let's place `require "open3"` at the top of the file (can be commented / uncommented, either works). That's what we've been doing with other languages, example: 

<img width="831" alt="Screenshot 2022-12-13 at 15 36 02" src="https://user-images.githubusercontent.com/3893573/207376580-ab9090e7-d5b1-47ec-b4fd-389251d01f59.png">

For context: the most common usage here is that a user will just blindly uncomment the block and then run tests. They'll _then_ see "Logs from your program will appear here!" in the test output and might remove that line in later stages.